docs/core/compatibility/2.2-3.0.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Breaking changes, Version 2.2 to 3.0
```

Comment:
```suggestion
title: Breaking changes, version 2.2 to 3.0
```

Reply:
Do you also want to add a suffix title to this folder to display the .NET Core brand?